Dockerfile

```
@@ -0,0 +1,19 @@
+FROM node:10-alpine
+
+ENV TZ=Europe/Zurich
+
+RUN adduser -D nib
+WORKDIR /home/nib
+
+COPY bin/ ./bin/
+COPY lib/ ./lib/
+COPY plugins/ ./plugins/
+COPY package.json package-lock.json* ./
+COPY index.js ./
+
+RUN chown -R nib:nib .
+USER nib
+
+RUN npm install
```

To better utilize the layered caches, I'd move some things around

```suggestion
COPY package.json package-lock.json ./
RUN npm install

COPY bin lib plugins index.js .

RUN chown -R nib:nib .
USER nib
```